pkgs/applications/networking/ids/zeek/script.nix

```diff
@@ -1,69 +1,12 @@
-{coreutils}:
+{ coreutils }:
 ''
-   sed -i 's|/bin/mv|${coreutils}/bin/mv|' scripts/base/frameworks/logging/writers/ascii.zeek
-   sed -i 's|/bin/mv|${coreutils}/bin/mv|' scripts/policy/misc/trim-trace-file.zeek
-   sed -i 's|/bin/cat|${coreutils}/bin/cat|' scripts/base/frameworks/notice/actions/pp-alarms.zeek
-   sed -i 's|/bin/cat|${coreutils}/bin/cat|' scripts/base/frameworks/notice/main.zeek
+  sed -i 's|/bin/rm|${coreutils}/bin/rm|' scripts/policy/misc/trim-trace-file.zeek
```

Comment:
If we use substituteInPlace we get a warning if the pattern does not match anything.

Reply:
@SuperSandro2000  please, review again. I think got all things in the right.
such as:
```
/nix/store/ypsd29c5hgj1x7xz5ddffanxw5d8fh7b-coreutils-8.32/bin/cat %s %s | %s -t -oi && /nix/store/yp
sd29c5hgj1x7xz5ddffanxw5d8fh7b-coreutils-8.32/bin/rm 
```